retrieveProducts/swagger-api.yaml

```diff
@@ -11,6 +11,8 @@ paths:
   /products:
     get:
       responses: {}
+    post:
+      responses: {}
     options:
```

Comment:
Why do we need a couple of options requests? 
![image](https://user-images.githubusercontent.com/24233570/98799549-b215d880-2428-11eb-97ab-96827e622132.png)

Reply:
this is default requests created by ApiGateway when you enable cors for some endpoints